package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/privacy-types",
   "description": "Core enums and types that can be useful when interacting with Transcend's public APIs.",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Is there a CI job to bump these numbers when changes are made?

Reply:
no there isn’t, but it will just fail to publish. i think it’s fine to do manually since someone will realize that they forgot to bump the version when they go to consume the changes.

we have a bunch of repos like this now and none of them have that check